lib/blemulator/simulation/simulated_characteristic.dart

```diff
@@ -36,18 +36,21 @@ class SimulatedCharacteristic {
 
   Future<void> write(Uint8List value) async {
     this._value = value;
-    if (_streamController?.hasListener == true)
-      _streamController.sink.add(value);
+    if (streamController?.hasListener == true)
+      streamController.sink.add(value);
   }
 
   Stream<Uint8List> monitor() {
-    if (_streamController == null) {
-      _streamController = StreamController(onCancel: () {
-        _streamController.close();
-        _streamController = null;
+    if (!isNotifiable) {
```

Comment:
It's not that easy, ask @Cierpliwy

Reply:
We decided to throw error when someone will try monitor characteristic that is neither notifiable nor indicatable.